extensions/spring-di/deployment/pom.xml

```diff
@@ -35,9 +35,9 @@
         </dependency>
 
         <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-context</artifactId>
-            <scope>test</scope>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-spring-di-api</artifactId>
+<!--            <scope>test</scope>-->
```

Comment:
This is weird.

The dependency should only be in the runtime artifact, shouldn't it?